src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -158,6 +160,15 @@ public function removeEventListener($events, $listener)
         $hash = $this->getHash($listener);
 
         foreach ((array) $events as $event) {
+            if (!isset($this->initialized[$event])) {
+                $this->initializeListeners($event);
```

Comment:
Only solution found so far for the `testRemoveLazyListenerInitialized`

Reply:
I wouldn't support this use case personally. Let's remove this part.

Reply:
I removed it then @nicolas-grekas